src/util/favorites.js

```diff
@@ -167,7 +167,7 @@ const models2objects = config => {
     return config;
 };
 
-const cleanDimension = dim => ({
+export const cleanDimension = dim => ({
```

Comment:
is this really located in the right place, now that it's being used by both `favourites` and `analyticalObject.js`?

Reply:
Yes, it could be moved to the util/analytics.js, but it requires a bigger rewrite I would like to save for later (and possibly reuse code you've made for DV).